src/main/kotlin/org/pkl/intellij/action/PklDownloadDependencySourcesAction.kt

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.pkl.intellij.action
+
+import com.intellij.codeInsight.intention.IntentionAction
+import com.intellij.collaboration.async.CompletableFutureUtil.handleOnEdt
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.Messages
+import com.intellij.psi.PsiFile
+import org.pkl.intellij.packages.PackageDependency
+import org.pkl.intellij.packages.pklPackageService
+import org.pkl.intellij.psi.enclosingModule
+import org.pkl.intellij.toolchain.pklCli
+
+class PklDownloadDependencySourcesAction : IntentionAction {
+  override fun startInWriteAction(): Boolean = false
+
+  override fun getText(): String = "Download dependencies"
```

Comment:
I don't mind either, but the naming divergence is a bit confusing. Why does the action name stipulate it concerns `Sources`, but the `Text` does not?

Reply:
I don't have a better answer than "it's the name I came up with". But, will change this name to "Download dependency sources"